lib/client/api_matrix.es6.js

```diff
@@ -11,12 +11,13 @@ foam.CLASS({
     to CSV format string or JSON matrix that can be displayed on HTML as
     a nested table.`,
   requires: [
-    'org.chromium.apis.web.Browser',
-    'org.chromium.apis.web.WebInterface',
-    'org.chromium.apis.web.BrowserWebInterfaceJunction',
-    'foam.dao.EasyDAO',
+    'foam.dao.AnonymousSink',
```

Comment:
[It's here](https://github.com/foam-framework/foam2/blob/073884a6e4e3900c48f4aebaf664447235dfd431/src/foam/dao/Sink.js#L181). It allows for providing a partial `Sink` implementation and fills in noops for other methods.